### PR TITLE
Ability to configure min frames for zone presence

### PR DESCRIPTION
**Pull request by NickM-27**
I made this work as a zone inertia where the object needs to be detected in the zone in quick succession to be considered in the zone.

### CODE REVIEW

File: frigate/config.py [hunk 1]
```diff
@@ -326,6 +326,12 @@ class ZoneConfig(BaseModel):
     coordinates: Union[str, List[str]] = Field(
         title="Coordinates polygon for the defined zone."
     )
+    inertia: int = Field(
+        default=3,
+        title="Number of frames required for object to be considered present in the zone.",
```
Comment (blakeblackshear) [inline frigate/config.py:331]:
```suggestion
        title="Number of consecutive frames required for object to be considered present in the zone.",
```

File: docs/docs/configuration/index.md [hunk 2]
```diff
@@ -467,6 +467,8 @@ cameras:
         # Required: List of x,y coordinates to define the polygon of the zone.
         # NOTE: Presence in a zone is evaluated only based on the bottom center of the objects bounding box.
         coordinates: 545,1077,747,939,788,805
+        # Optional: Set the inertia required for an object to be considered present in the zone. Allowed values are 0 < inertia < 10 (default: shown below)
```
Comment (blakeblackshear) [inline docs/docs/configuration/index.md:470]:
```suggestion
        # Optional: Number of consecutive frames required for object to be considered present in the zone. Allowed values are 1-10 (default: shown below)
```

**Review by blakeblackshear — CHANGES_REQUESTED** [review]

**Review by blakeblackshear — APPROVED** [review]

**Comment by Inrego** [comment]
Any idea when this will be merged into latest? I don't know much about the release cycle of Frigate.

**Comment by Inrego** [comment]
Is there a roadmap or similar where I can see what's planned for 0.13 to get an idea of the progress towards that goal?

**Comment by NickM-27** [comment]
https://github.com/blakeblackshear/frigate/pull/6262

**Pull request merged**